images/style.css

```diff
@@ -0,0 +1,379 @@
+
+.body{
+    background-color: rgb(234, 231, 231);
+    display: grid;
+    grid-template-columns: 1fr 3fr 1fr;
+    font-family: Arial, Helvetica, sans-serif;
+    margin: 10px 10px;
+    /* margin-left: min(500px);
+    margin-right: min(500px); */
+}
+
+h1{
+    font-weight: bold;
+    font-size: 15px;
+}
+
+h2{
+    font-weight: bold;
+    font-size: 15px;
+}
+
+h3{
+    font-weight: bold;
+    font-size: 12px;
+}
+
+p{
+    font-size: 12px;
+}
+
+strong{
+    font-weight: bold;
+    font-size: 12px;
+}
+
+span{
+    font-size: 12px;
+}
+
+.left-menu-bar-section{
+    padding-bottom: 100px;
+    background-color: rgb(245, 245, 245);
+}
+
+.left-menu-bar-section img{
+    height: 30px;
+    width: 30px;
+    padding: 5px 10px 5px 5px;
+ }
+
+ .logo{
+    filter: invert(57%) sepia(10%) saturate(2812%) hue-rotate(83deg) brightness(89%) contrast(86%);
+ }
+
+ .logo:hover{
+    filter: invert(97%) sepia(0%) saturate(0%) hue-rotate(238deg) brightness(87%) contrast(96%);
+ }
+
+.left-menu-bar-section > .left-menu-header{
+    padding-top: 5px;
+    padding-left: 20px;
+    height: 40px;
+    object-fit: contain;
+}
+
+.left-menu-bar-section .nav{
+    display: grid;
+    grid-template-rows: repeat(7, 1fr);
+    padding: 20px;
+    gap: 5px;
+    justify-items: left; 
+    align-items: center;
+    margin-bottom: 50px;
+}
+
+.left-menu-bar-section .nav span{
+    font-weight: bold;
+
+}
+
+.left-menu-bar-section .nav .home:hover{
```

Comment:
Interesting way of setting and changing the colors, definitely need to play around with this myself.